src/components/movies/MovieList.vue

```diff
@@ -20,19 +20,31 @@ export default {
     MovieCard,
   },
   data() {
-    return {};
+    return {
+      selectedGenre: "",
```

Comment:
Maybe it's a good idea to assign "All movies" as default to selectedGenre, because there is ultimately some value for select genres...

Reply:
Thank you Kasia, that. makes so much more sense ❤️